unittests/Support/CommandLineTest.cpp

```diff
@@ -72,12 +72,12 @@ class StackOption : public cl::opt<T> {
 
 cl::OptionCategory TestCategory("Test Options", "Description");
 TEST(CommandLineTest, ModifyExisitingOption) {
-  StackOption<int> TestOption("test-option", cl::desc("old description"));
-
   const char Description[] = "New description";
   const char ArgString[] = "new-test-option";
   const char ValueString[] = "Integer";
 
+  StackOption<int> TestOption("test-option", cl::desc("old description"));
```

Comment:
Can you wrap the block from the top of this test function down to below the option declaration in a `//HLSL Change begin/end` comment block?

Reply:
Done.